spec/index.bs

```diff
@@ -4049,6 +4076,22 @@ enum GPUStencilOperation {
 };
 </script>
 
+<div algorithm>
+    <dfn abstract-op>validating GPUDepthStencilStateDescriptor</dfn>(descriptor)
+    **Arguments:**
+        - {{GPUDepthStencilStateDescriptor}} |descriptor|
+
+    Return `true`, if and only if, all of the following conditions are satisfied:
+
+        - |descriptor|.{{GPUDepthStencilStateDescriptor/format}} is listed in {#depth-formats}.
```

Comment:
nit: this doesn't handle extensions that enable formats like `stencil8`

Reply:
we do have `stencil8` in the spec :)
But yes, this doesn't handle extensions. I'll add an inline issue.